### ISSUE TITLE
remote: Permission to cui-unige/modeling-verification-1.git denied to beny3.

### ISSUE THREAD

**Issue by beny3** [cui-unige/petri-nets-beny3#1]
Bonjour, je n'arrive pas as pusher, a priori je n'ai pas la permission

remote: Permission to cui-unige/modeling-verification-1.git denied to beny3.

devez vous me la donner?

merci

**Comment by saucisson** [comment]
Non, tu as bien la permission en écriture sur ton dépôt. Là tu essaye de faire un push sur mon dépôt à moi et non le tiens. Ton dépôt est à l'url : https://github.com/cui-unige/petri-nets-beny3.git

**Comment by beny3** [comment]
ok ca marche merci

**Comment by beny3** [comment]
J'ai implémenté tous les TODOs mais je n'ai pas écrit de test pour farkas, pour vérifier j'affiche la matrice réduite par Farkas dans la console, il s'agit d'un test visuel 

Merci, bonne journée

**Comment by saucisson** [comment]
Un test qui nécessite que quelqu'un regarde n'est pas un vrai test.
